src/components/sidenav/sidenav.spec.ts

```diff
@@ -111,7 +110,8 @@ export function main() {
         let sidenavElement = fixture.debugElement.query(By.css('md-sidenav'));
         let sidenavBackdropElement = fixture.debugElement.query(By.css('.md-sidenav-backdrop'));
         expect(getComputedStyle(sidenavElement.nativeElement).visibility).toEqual('visible');
-        expect(getComputedStyle(sidenavBackdropElement.nativeElement).visibility).toEqual('visible');
+        expect(getComputedStyle(sidenavBackdropElement.nativeElement)
```

Comment:
If this is only a matter of line length, could you split after `.visibility`? it looks better like this:

```
expect(getComputedStyle(sidenavBackdropElement.nativeElement).visibility)
    .toEqual('visible');
```